source/scss/pl-specific/styleguide.min.scss

```diff
@@ -2,6 +2,5 @@
     $STYLEGUIDE CHROME (WRAPPER)
 \*------------------------------------*/
 
-@import "settings.variables";
```

Comment:
what is this file? why is there a minified `.scss` file in the source? I would only expect `.css` files to be minified.

Reply:
This is to style the Pattern Lab-specific styles, e.g. navigation, annotations, pattern info drawer. This will not be something that gets deployed to a production environment.